src/main/java/mate/academy/exception/RegistrationException.java

```diff
@@ -0,0 +1,6 @@
+package mate.academy.exception;
+
+public class RegistrationException extends Exception {
+    public RegistrationException(String s) {
```

Comment:
The constructor should pass the message `s` to the superclass `Exception` to ensure the exception message is properly stored and accessible. Use `super(s);` within the constructor.